app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+  belongs_to :author, class_name: 'User'
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_the_post_counter
+    author.update(postscounter: author.postscounter + 1)
+  end
```

Comment:
- [ ] It would be nice if we use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then we make the `update_post_counter` method private